nx/lib/nx.ex

```diff
@@ -1609,6 +1639,14 @@ defmodule Nx do
     backend.constant(out, number, options)
   end
 
+  def to_tensor(bool) when is_boolean(bool) do
```

Comment:
We don't want this change either. :) The goal is to only allow it in `Nx.tensor`.